docker/l2s/src/index.ts

```diff
@@ -33,6 +34,7 @@ type BlockTime = {
 
 const CMC_ETH_ID = 1027;
 const DEFAULT_DECIMAL = 18;
+const DEFAULT_CHUNK_SIZE = 50;
```

Comment:
@viettiennguyen029 make it to ENV variable to able to change without code change